circle.yml

```diff
@@ -0,0 +1,12 @@
+dependencies:
+  pre:
+    - wget https://github.com/coreos/etcd/releases/download/v0.4.6/etcd-v0.4.6-linux-amd64.tar.gz
+    - tar xzvf etcd-v0.4.6-linux-amd64.tar.gz
+    - sudo cp etcd-v0.4.6-linux-amd64/etcd* /usr/local/bin/
+  override:
+    - 'rvm-exec 2.1.5 bundle install'
```

Comment:
As we didn't specify the ruby version in the Gemfile it uses ruby 1.9.3. As I didn't know if this was on purpose, and I am not sure if you can use a gem written with ruby 2.2.x with no problems in a program using ruby 2.1.5 I chose to do the tests with the same version as we have for the backend